config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
+  newName: quay.io/mmascia/vault-config-operator
```

Comment:
this is not right and if you felt you needed to do it there is something wrong

Reply:
no forgot to roll back it.